compiler+runtime/test/jank/reader-macro/cstring/fail-missing-string-cstring.jank

```diff
@@ -0,0 +1 @@
+#cpp
```

Comment:
Let's move these to `test/jank/reader-macro/cpp/string/`, since the reader macro is `cpp` and the value that follows is a string. That way we can group all `#cpp` types under that same directory.